spec/graphlient/client_query_spec.rb

```diff
@@ -91,7 +91,7 @@
         expect do
           client.execute(query, id: '42')
         end.to raise_error Graphlient::Errors::GraphQLError do |e|
-          expect(e.to_s).to eq 'Variable id of type Int was provided invalid value'
+          expect(e.to_s).to eq 'Variable $id of type Int was provided invalid value'
```

Comment:
This is due to allowing a newer graphql version, which was required for Ruby 3.0-compatibility.